tests/test_check_datapackage.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from data_checks.check_datapackage import run_checks
+
+
+class TestCheckDatapackage(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def test_perfect_datapackage(self):
+        file_path = 'tests/datapackages/2018-19/epre/perfect_datapackage.json'
```

Comment:
what I had in mind here was actually for the tests to be in paths like `test/epre_perfect_datapackage/2018-19/epre` so then you run `python bin/data-checks.py test/epre_perfect_datapackage` then it's easy to run negative examples cases while developing schemas. And this suite is just there to run them all and ensure they pass or fail appropriately. And the interface of `{{ financial_year}}/{{ dataset_type }}/datapackage.json` doesn't get violated